check_process

```diff
@@ -6,7 +6,7 @@
 		path="/gitlab"	(PATH)
 		admin="john"	(USER)
 		is_public=1	(PUBLIC|public=1|private=0)
-		use_web_account=0
+		use_web_account=0 (WEB_ACCOUNT)
```

Comment:
What does that mean ?
Don't think package check will use that, not sure either it's not going to interfere.
Anyway, looks ok https://ci-apps-hq.yunohost.org/jenkins/job/gitlab_ynh%20PR61/1/console